src/turnkeyml/build/stage.py

```diff
@@ -329,6 +342,11 @@ def launch(self, state: build.State) -> build.State:
 
             return state
 
+        finally:
+            # Collect telemetry about the stage
+            execution_time = time.time() - start_time
+            stats.save_model_eval_stat(stage.duration_key, execution_time)
```

Comment:
Nice! I can see how this fixes issues when errors happen. Now we will know the elapsed time of stages that errored out. This is fine, since we are not calling the report column "completed stages" anymore.

Reply:
no action needed here :)